### PR TITLE
PEM-4742-Correct documentation for edb_wait_states in Performance Diagnostic topic

### PR DESCRIPTION
**Pull request by arup-roy2023**
## What Changed?

Prerequisites first bulleted point content change. The link to 'EDB wait states' in EPAS doc is provided.  This is part of the Performance Diagnostic topic in both doc and OLH.

This change is based on https://enterprisedb.atlassian.net/browse/PEM-4742

### CODE REVIEW

File: product_docs/docs/pem/9/pem_online_help/04_toc_pem_features/21_performance_diagnostic.mdx [hunk 1]
```diff
@@ -20,7 +20,7 @@ You can analyze the Wait States data on multiple levels by narrowing down your s
 
 Prerequisite:
 
--   For PostgreSQL, you need to install `edb_wait_states_<X>` package from `edb.repo` where `<X>` is the version of PostgreSQL Server. You can refer to [EDB Build Repository](https://repos.enterprisedb.com/) for the steps to install this package. For Advanced Server, you need to install `edb-as<X>-server-edb-modules`, Where `<X>` is the version of Advanced Server.
+-   For PostgreSQL, you need to install `edb_wait_states_<X>` package from `edb.repo` where `<X>` is the version of PostgreSQL Server. You can refer to [EDB Build Repository](https://repos.enterprisedb.com/) for the steps to install this package. For Advanced Server, you must install EDB wait states; see [EDB Wait States Background Worker (EWSBW)](/epas/latest/epas_guide/13_performance_analysis_and_tuning/#edb-wait-states-background-worker-ewsbw) under [EDB wait states](/epas/latest/epas_guide/13_performance_analysis_and_tuning/#edb-wait-states).
```
Comment (sonotley) [inline product_docs/docs/pem/9/pem_online_help/04_toc_pem_features/21_performance_diagnostic.mdx:23]: What is `edb.repo`? some kind of dynamic variable? It was there before so I assume it's fine.

**Review by sonotley — DISMISSED** [review]

File: product_docs/docs/pem/9/pem_online_help/04_toc_pem_features/21_performance_diagnostic.mdx (hunk 1 above)
Comment (nidhibhammar) [inline product_docs/docs/pem/9/pem_online_help/04_toc_pem_features/21_performance_diagnostic.mdx:23]:
```suggestion
-   For PostgreSQL, you need to install `edb_wait_states_<X>` package from `edb.repo` where `<X>` is the version of PostgreSQL Server. To install this package, you can refer to [EDB Build Repository](https://repos.enterprisedb.com/). For Advanced Server, you must install EDB wait states; see  [EDB wait states](/epas/latest/epas_guide/13_performance_analysis_and_tuning/#edb-wait-states).
```

File: product_docs/docs/pem/9/tuning_performance/performance_diagnostic.mdx [hunk 2]
```diff
@@ -21,7 +21,7 @@ To analyze the Wait States data on multiple levels, narrow down the data you sel
 
 ## Prerequisites
 
-- For PostgreSQL, you need to install the `edb_wait_states_<X>` package from `edb.repo`, where `<X>` is the version of PostgreSQL Server. For the steps to install this package, see [EDB Build Repository](https://repos.enterprisedb.com/). For EDB Postgres Advanced Server, you need to install the `edb-as<X>-server-edb-modules`, where `<X>` is the version of EDB Postgres Advanced Server.
+- For PostgreSQL, you need to install the `edb_wait_states_<X>` package from `edb.repo`, where `<X>` is the version of PostgreSQL Server. For the steps to install this package, see [EDB Build Repository](https://repos.enterprisedb.com/). For EDB Postgres Advanced Server, you must install EDB wait states; see [EDB Wait States Background Worker (EWSBW)](/epas/latest/epas_guide/13_performance_analysis_and_tuning/#edb-wait-states-background-worker-ewsbw) under [EDB wait states](/epas/latest/epas_guide/13_performance_analysis_and_tuning/#edb-wait-states).
```
Comment (nidhibhammar) [inline product_docs/docs/pem/9/tuning_performance/performance_diagnostic.mdx:24]:
```suggestion
-   Install the EDB wait states package:
     
     -  For PostgreSQL, see [EDB Build Repository](https://repos.enterprisedb.com/). 
     
     -  For Advanced Server, see  [EDB wait states](/epas/latest/epas_guide/13_performance_analysis_and_tuning/#edb-wait-states).
```

**Review by nidhibhammar — COMMENTED** [review]
Have left my suggestions on the changes

File: product_docs/docs/pem/9/pem_online_help/04_toc_pem_features/21_performance_diagnostic.mdx (hunk 1 above)
Comment (arup-roy2023) [reply product_docs/docs/pem/9/pem_online_help/04_toc_pem_features/21_performance_diagnostic.mdx]: I have updated the content.

File: product_docs/docs/pem/9/tuning_performance/performance_diagnostic.mdx (hunk 2 above)
Comment (arup-roy2023) [reply product_docs/docs/pem/9/tuning_performance/performance_diagnostic.mdx]: I have updated the content

File: product_docs/docs/pem/9/pem_online_help/04_toc_pem_features/21_performance_diagnostic.mdx [hunk 3]
```diff
@@ -20,7 +20,11 @@ You can analyze the Wait States data on multiple levels by narrowing down your s
 
 Prerequisite:
 
--   For PostgreSQL, you need to install `edb_wait_states_<X>` package from `edb.repo` where `<X>` is the version of PostgreSQL Server. You can refer to [EDB Build Repository](https://repos.enterprisedb.com/) for the steps to install this package. For Advanced Server, you need to install `edb-as<X>-server-edb-modules`, Where `<X>` is the version of Advanced Server.
+- Install the EDB wait states package:
+     
+  - For PostgreSQL, see [EDB Build Repository](https://repos.enterprisedb.com/).
```
Comment (nidhibhammar) [inline product_docs/docs/pem/9/pem_online_help/04_toc_pem_features/21_performance_diagnostic.mdx:25]:
```suggestion
  - For PostgreSQL, see [EDB repository](https://repos.enterprisedb.com/). 
```

File: product_docs/docs/pem/9/tuning_performance/performance_diagnostic.mdx [hunk 4]
```diff
@@ -21,7 +21,11 @@ To analyze the Wait States data on multiple levels, narrow down the data you sel
 
 ## Prerequisites
 
-- For PostgreSQL, you need to install the `edb_wait_states_<X>` package from `edb.repo`, where `<X>` is the version of PostgreSQL Server. For the steps to install this package, see [EDB Build Repository](https://repos.enterprisedb.com/). For EDB Postgres Advanced Server, you need to install the `edb-as<X>-server-edb-modules`, where `<X>` is the version of EDB Postgres Advanced Server.
+- Install the EDB wait states package:
+     
+    - For PostgreSQL, see [EDB Build Repository](https://repos.enterprisedb.com/).
```
Comment (nidhibhammar) [inline product_docs/docs/pem/9/tuning_performance/performance_diagnostic.mdx:26]:
```suggestion
    - For PostgreSQL, see [EDB repository](https://repos.enterprisedb.com/). 
```

File: product_docs/docs/pem/9/pem_online_help/04_toc_pem_features/21_performance_diagnostic.mdx (hunk 3 above)
Comment (arup-roy2023) [reply product_docs/docs/pem/9/pem_online_help/04_toc_pem_features/21_performance_diagnostic.mdx]: Done

File: product_docs/docs/pem/9/tuning_performance/performance_diagnostic.mdx (hunk 4 above)
Comment (arup-roy2023) [reply product_docs/docs/pem/9/tuning_performance/performance_diagnostic.mdx]: Done

**Review by nidhibhammar — APPROVED** [review]

**Pull request merged**